src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -254,6 +245,17 @@ class Meta:
         extra_kwargs = {
             "url": {"lookup_field": "uuid"},
             "uuid": {"read_only": True},
+            "zaaktype": {
+                "lookup_field": "uuid",
+                "max_length": 1000,
+                "min_length": 1,
+                "validators": [
+                    LooseFkResourceValidator("ZaakType", settings.ZTC_API_SPEC),
+                    LooseFkIsImmutableValidator(),
+                    PublishValidator(),
+                ],
+                "help_text": get_help_text("zaken.Zaak", "zaaktype"),
```

Comment:
this sould already happen by default, I think this line can be removed

Reply:
`help_text` is removed